nss-day-cohort-48/rare-server-boastful-bulldogs
# Added SQL queries for posts

Added queries for all posts, single post and post by userId

## Steps to Test
1. git fetch --all
2. git checkout anm-get-all-posts
3. run debugger and serve client
4. verify posts populate on client
